lib/travis/build/addons/apt.rb

```diff
@@ -126,7 +126,7 @@ def add_apt_packages
 
             unless whitelisted.empty?
               sh.export 'DEBIAN_FRONTEND', 'noninteractive', echo: true
-              sh.cmd "sudo -E apt-get -yq update &>> ~/apt-get-update.log", echo: true, timing: true
+              sh.cmd "sudo -E apt-get -yq update", echo: true, timing: true
```

Comment:
I suspect not, unfortunately. That's another part of why https://github.com/travis-ci/apt-source-whitelist/issues/156 took me so long to figure out: `apt-get update` failed, and instead of failing the build immediately it continued on, to produce cryptic errors later.